Documentation/MigrateToComposer/MigrationSteps.rst

```diff
@@ -122,7 +122,39 @@ You already know the TER and always used it to install extensions? Fine.
 But with composer, the **preferred way** is to install extensions
 directly from `packagist.org <https://packagist.org>`__. This works great, when the maintainer uploaded them
 to there. Many well known extensions are already available.
-You only need to known the package name. There are multiple ways to find it.
+You only need to known the package name. There are multiple ways to find it:
+
+Notice on extension's TER page
```

Comment:
You can use correct title capitalization, as recently documented: https://docs.typo3.org/typo3cms/HowToDocument/GeneralConventions/ContentStyleGuide.html#rules-for-title-case This is consistently used in most of this guide and should be changed in rest of chapter as well.